users/usersControllers.js

```diff
@@ -0,0 +1,56 @@
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const User = require('./usersModels');
+
+exports.signup = async (req, res, next) => {
```

Comment:
This returns a promise so adding next() is not necessary since it's not a middleware